cvat/apps/engine/views.py

```diff
@@ -81,8 +83,11 @@ def get_serializer(self, *args, **kwargs):
         pass
 
     @staticmethod
-    @swagger_auto_schema(method='get', operation_summary='Method provides basic CVAT information',
-        responses={'200': AboutSerializer})
+    @extend_schema(summary='Method provides basic CVAT information',
+        responses={
```

Comment:
- due to giving `serializer_class=AboutSerializer`, the serializer is detected automatically.
- 200 is the default code for actions (for native viewsets methods (list/retrieve/etc) 200, 201, 204 are detected automatically)

so this `responses=...` is not needed. spectacular gets this right automatically. you would only need it if `many=True` were used, like in some other places.